sdk/src/main/java/com/silanis/esl/sdk/examples/PackageReferencedConditionsExample.java

```diff
@@ -0,0 +1,139 @@
+                        .build())
+                )
+            )
+            .withDocument(newDocumentWithName("PackageReferencedConditionsExampleDocument2")
+                .fromStream(documentInputStream2, DocumentType.PDF)
+                .withId(DOCUMENT_ID_2)
+                .withSignature(signatureFor(email1)
+                    .onPage(0)
+                    .atPosition(100, 400)
+                    .withId(signatureId2)
+                    .withField(FieldBuilder.textField()
+                        .withName("field4")
+                        .withId(fieldId4)
+                        .atPosition(400, 100)
+                        .onPage(0)
+                        .build())
+                    .withField(FieldBuilder.textField()
+                        .withName("field5")
+                        .withId(fieldId5)
+                        .atPosition(400, 200)
+                        .onPage(0)
+                        .build())
+                    .withField(FieldBuilder.textField()
+                        .withName("field6")
+                        .withId(fieldId6)
+                        .atPosition(400, 300)
+                        .onPage(0)
+                        .build())
+                )
+            )
+            .withCondition(createCondition("condition1", DOCUMENT_ID_1, fieldId1.getId(), DOCUMENT_ID_1, fieldId2.getId()))
+            .withCondition(createCondition("condition2", DOCUMENT_ID_1, fieldId1.getId(), DOCUMENT_ID_1, fieldId3.getId()))
+            .withCondition(createCondition("condition3", DOCUMENT_ID_2, fieldId4.getId(), DOCUMENT_ID_2, fieldId5.getId()))
+            .build();
+
+        packageId = eslClient.createPackageOneStep(superDuperPackage);
+
+        packageLevelRefConditions = eslClient.getPackageService().getReferencedConditions(packageId.getId());
+        documentLevelRefConditions = eslClient.getPackageService().getReferencedConditions(packageId.getId(), DOCUMENT_ID_1);
+        fieldLevelRefConditions = eslClient.getPackageService().getReferencedConditions(packageId.getId(), DOCUMENT_ID_1, fieldId1.getId());
+    }
+
+    private FieldCondition createCondition(String id, String conditionDocId, String conditionFieldId, String actionDocId, String actionFieldId) {
+        FieldCondition condition = new FieldCondition();
+        condition.setId(id);
+        condition.setCondition(String.format("document['%s'].field['%s'].empty == true", conditionDocId, conditionFieldId));
+        condition.setAction(String.format("document['%s'].field['%s'].enabled = true", actionDocId, actionFieldId));
+        return condition;
+    }
+
+    ReferencedConditions getPackageLevelRefConditions() {
+        return packageLevelRefConditions;
+    }
+
+    ReferencedConditions getDocumentLevelRefConditions() {
+        return documentLevelRefConditions;
+    }
+
+    ReferencedConditions getFieldLevelRefConditions() {
+        return fieldLevelRefConditions;
+    }
```

Comment:
This is an example. ㄲemove the above 3 private methods.

Reply:
Updated